### PR TITLE
analysis fixes

### PR DESCRIPTION
**Pull request by meganbkratz**
Fixes for loading user configurations, and opening Photostim module (fixes for Spinbox.setOpts change)

### CODE REVIEW

File: acq4/Manager.py [hunk 1]
```diff
@@ -415,7 +415,7 @@ def loadDefinedConfig(self, name):
             if name not in self.config['configurations']:
                 raise Exception("Could not find configuration named '%s'" % name)
             cfg = self.config['configurations'].get(name, )
-        self.configure(cfg)
+        self.configure({'storageDir':cfg})
```
Comment (campagnola) [inline acq4/Manager.py]: I din't think this is right--shouldn't `cfg` already be a dict with `storageDir` inside?

File: acq4/pyqtgraph/widgets/SpinBox.py [hunk 2]
```diff
@@ -143,7 +143,7 @@ def setOpts(self, **opts):
         """
         #print opts
         for k in opts:
-            if k == 'bounds':
+            if k == 'bounds' or k == 'range':
```
Comment (campagnola) [inline acq4/pyqtgraph/widgets/SpinBox.py]: Why? I don't think `range` was a valid option here in any version of pyqtgraph.

File: acq4/Manager.py (hunk 1 above)
Comment (meganbkratz) [inline acq4/Manager.py]: You're right, I had my config file set up incorrectly. I changed the example config file to make it more clear how to set things up.

File: acq4/pyqtgraph/widgets/SpinBox.py (hunk 2 above)
Comment (meganbkratz) [inline acq4/pyqtgraph/widgets/SpinBox.py]: I think you're right, but there are a lot of places (like in all the flowchart nodes) where we expect 'range' to be a valid option. These all probably failed silently in the past. Adding 'range' as an option to SpinBox seemed like a reasonable thing. Alternatively, we could change all the places we pass in range to use bounds instead. Thoughts?

Comment (campagnola) [inline acq4/pyqtgraph/widgets/SpinBox.py]: Let's just fix range->bounds wherever we see it. I'd prefer to keep the SpinBox API clean.

**Comment by meganbkratz** [comment]
Hey Luke, 
Could you revisit merging this in? Thanks!
Megan

**Comment by campagnola** [comment]
Thanks!

**Pull request merged**